lib/db/schema/patch-039-040.sql

```diff
@@ -0,0 +1,66 @@
+ALTER TABLE `verificationReminders` DROP PRIMARY KEY, ADD PRIMARY KEY (`uid`, `type`);
```

Comment:
I'm guessing that we can't use `ALGORITHM = INPLACE, LOCK = NONE` on this `ALTER TABLE`, meaning that it will block any other attempts to write into this table.  Does locking this table matter in production?  If e.g. we're not writing anything into the table yet in production then it's probably OK, but may be worth a short comment to that effect in the migration here.